### PR TITLE
Forms Completion

### PR DESCRIPTION
**Pull request by RNtaate**
## What does this PR do?
- [X] Creates a User model.
- [X] Validates the username, email, and password for the users table.
- [X]  Creates a form template for creating a new user.
- [X] Creates a form template for editing an existing user.
- [X] Fixes rubocop and stylelint offenses.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<h1>Create new user</h1>
+<ul>
+  <% @user.errors.full_messages.each do |error| %>
+    <li><%= error %></li>
+  <% end %>
+</ul>
+
+<% if false %>
+
+  <form accept-charset="UTF-8" action="/users" method="POST">
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Please use `users_path` from rails helpers

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <label for="password">password:</label>
+    <input type="password" id="password" name="user[password]"><br><br>
+    <label for="confirmpass">confirm password:</label>
+    <input type="password" id="confirmpass" name="user[password_confirmation]"><br><br>
+    <input type="submit" value="Create">
+  </form>
+<% end %>
+
+<% if false%>
+  <%= form_tag("/users", method: "POST") do %>
```
Comment (maelfosso) [inline app/views/users/new.html.erb]: Please use `users_path` from rails helpers

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username],
+    #                  email: params[:email],
+    #                  password: params[:password],
+    #                  password_confirmation: params[:password_confirmation])
```
Comment (maelfosso) [inline app/controllers/users_controller.rb]: Kindly remove comments

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

You did a good job
Kindly read the comments left below, make the suggested changes.
And submit a new code review request

Thanks

Mael FOSSO

**Comment by RNtaate** [comment]
## Changes made
- [X] Changed action value in the first and second forms to users_path in new.html.erb file.
- [X] Removed unwanted comments from the users_controller.rb file.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED 🍾 
Excellent job on this project guys. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job :+1:
- You can now merge these changes.
- I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**